srcpkgs/telegram-desktop/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'telegram-desktop'
 pkgname=telegram-desktop
-version=4.1.1
-revision=1
+version=4.2.4
+revision=2
```

Comment:
Revision should be reset (or kept) at 1 for every new version, it only needs to be increased when rebuilding without changing the version

Reply:
Thank you for your quick answer. I fixed it, sorry for the error.